app/data_model/answer_store.py

```diff
@@ -247,6 +254,26 @@ def get_hash(self):
         """
         return hash(json.dumps(self.answers, sort_keys=True))
 
+    def upgrade(self):
```

Comment:
I think we need to consider how we manage multiple versions and upgrades of answers across multiple version changes e.g. loading data v1 into data v3. We also need to consider how we test these migrations.